index.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] It will be nice if you could have please follow the previous reviewer's suggestion which was to enable linters to check on your code. Please remove these lines.